@AH-64D Apache Longbow/Addons/fza_ah64_ase/functions/fn_init.sqf

```diff
@@ -17,26 +17,90 @@ Examples:
 Author:
     Snow(Dryden)
 ---------------------------------------------------------------------------- */
+#include "\fza_ah64_ase\headers\ase.h"
 params ["_heli"];
 
 if (!(_heli getVariable ["fza_ah64_aseInitialised", false]) && local _heli) then {
     _heli setVariable ["fza_ah64_aseInitialised", true, true];
 
+    _heli setVariable ["fza_ah64_ase_updateTimer",      0,                      true];
+
     //--Chaff       - Aircraft common
-    _heli setVariable ["fza_ah64_ase_chaffState",       "safe", true];
+    _heli setVariable ["fza_ah64_ase_chaffState",       ASE_CHAFF_STATE_SAFE,   true];
     //--RLWR        - Aircraft Common
-    _heli setVariable ["fza_ah64_ase_rlwrPwr",          "off",  true];
-    _heli setVariable ["fza_ah64_ase_rlwrCount",        0,      true];
-    _heli setVariable ["fza_ah64_ase_rlwrObjects",      [],     true];
-    _heli setVariable ["fza_ah64_ase_audioList",        [],     true];
+    _heli setVariable ["fza_ah64_ase_rlwrPwr",          ASE_RLWR_STATE_OFF,     true];
+    _heli setVariable ["fza_ah64_ase_rlwrObjects",      [],                     true];
     //--IR Jammer   - Aircraft Common
-    _heli setVariable ["fza_ah64_ase_msnEquipPwr",      "off",  true];
-    _heli setVariable ["fza_ah64_ase_irJamState",       0,      true];
-    _heli setVariable ["fza_ah64_ase_irJamTimer",       0.0,    true];
-    _heli setVariable ["fza_ah64_ase_irJamLaunchTimer", 0,      true];
+    _heli setVariable ["fza_ah64_ase_msnEquipPwr",      ASE_MSNEQUIP_STATE_OFF, true];
+    _heli setVariable ["fza_ah64_ase_irJamState",       [0,0,0],                true];
     //--Autopage    - Crewstation independent
-    _heli setVariable ["fza_ah64_ase_autopage",         1,      true];
-    _heli setVariable ["fza_ah64_ase_searchingObj",     [],     true];
-    _heli setVariable ["fza_ah64_ase_acquisitionObj",   [],     true];
-    _heli setVariable ["fza_ah64_ase_trackingobj",      [],     true];
-};
\ No newline at end of file
+    _heli setVariable ["fza_ah64_ase_autopage",         ASE_AUTOPAGE_OFF,       true];
+    _heli setVariable ["fza_ah64_ase_data",             [[],[],[]],             true];
+
+    _heli setVariable ["fza_ah64_adaDataBase", 
+    //Generic radar
+    [ ["B_SAM_SYSTEM_01_F",["RADAR", "SR"]],["B_SAM_SYSTEM_02_F",["RADAR", "SR"]]
```

Comment:
This seems like the wrong place for this data. It should be in Arma Cfg.

Maybe `FzaCfgADA` or similar?